### ISSUE TITLE
Inoperative Watch<-Debug on Windows-7-64 bit 

### ISSUE THREAD

**Issue by svmizin** [LADSoft/OrangeC#375]
Dear OCC creators.
Debug->Watch is absolutely disabled on my Windows-7-64 comp for any projects and any variables (global and locals) for both recent OCC versions : 6.0. 41.1 & 6.0.37.1 
I have some pause in programming, and when i started new project, i revealed that Watcher of debugger is absolutely Inoperative. Look at attached on-screen. When i am truing insert  ANY variable (global or local) in ANY watcher-window - i always get error message :"Undefined symbol".  Such situation i have now for ALL  right projects for Orange C:  for sample-projects , involved in OCC Installation Pack as far as all my own projects.
Also i attach one my own frame-project.
I will be glad and grateful for any your help.
[sum_gamma.zip](https://github.com/LADSoft/OrangeC/files/3482222/sum_gamma.zip)

![No_watch](https://user-images.githubusercontent.com/35298943/62711677-27168900-ba02-11e9-8a27-28466b94e84d.jpg)

**Comment by svmizin** [comment]
I apologize : **normal working of Watcher<-Debugger is restored, after changing OCC 6.0.37. for 6.0.41 and next many times rebuildings of all project.** 
Before that, i spent two day in unsuccessful  efforts to run Watcher.
**While i do not close this issue, checking REAL vanishment of the "bug"! 
Thank you.** 
Later i will close this issue if problem will not resume...
Besides that, some times ago i  found one more stable small defect, relating to the resource workshop...Therefore now i start a new one issue.

**Comment by LADSoft** [comment]
this seems to work in the current version; I looked in previous versions release notes and it looks like along about Version 6.0.38 there were a variety of bug fixes to OCIDE that were indicative of serious problems.   There was a lot of work at that time to fix the issues so it is probable this is why it didn't work in 6.0.37...   you would have had to do a 'rebuild all' with 6.0.38 or later to get it to work again.

I'll leave it for you to close whenever you are comfortable with it...

**Comment by svmizin** [comment]
Thank you very much, Dear David, thank you very much again !
You keep your responsibility and quality marks of your works on highest degree !
How i understad you, as i too old programmer... Having this heavy trouble with debugger,
at last i have guessed out, that was a new trouble, and changed OCC version, despite
i very do NOT like it during new project proceeding... 
With my best wishes, 
Sergey Mizin, at www.lebedev.ru




s.v.mizin на gmail.com

From: David Lindauer
Date: 2019-08-11 21:34
To: LADSoft/OrangeC
CC: svmizin; Author
Subject: Re: [LADSoft/OrangeC] Inoperative Watch<-Debug on Windows-7-64 bit (#375)
this seems to work in the current version; I looked in previous versions release notes and it looks like along about Version 6.0.38 there were a variety of bug fixes to OCIDE that were indicative of serious problems. There was a lot of work at that time to fix the issues so it is probable this is why it didn't work in 6.0.37... you would have had to do a 'rebuild all' with 6.0.38 or later to get it to work again.
I'll leave it for you to close whenever you are comfortable with it...

**Comment by GitMensch** [comment]
@svmizin Can you please recheck if this is solved with the current release (even if you are using an older one because of the other reported bug)?

**Comment by svmizin** [comment]
Yes, on Monday i will confirm, that in current OCC versions that bug is absent




s.v.mizin на gmail.com

From: Simon Sobisch
Date: 2019-08-23 14:56
To: LADSoft/OrangeC
CC: svmizin; Mention
Subject: Re: [LADSoft/OrangeC] Inoperative Watch<-Debug on Windows-7-64 bit (#375)
@svmizin Can you please recheck if this is solved with the current release (even if you are using an older one because of the other reported bug)?

**Comment by svmizin** [comment]
Dear comrades.
Thank you for your efforts. Now I deeply work with my new project on OCC. At that, I return to OCC v. 6.0.4**1**.1, because of volatile fails of 4**2**.1 version after 2nd dialog font changing. 
Now, in general, **debugger of OCC works good with one strange volatile fault, which I can not localize the while. It appears often if I put debugger break-down point BEFORE starting the program in debugger. In this case often, on unknown reason, the debugger do not catch this break-down point miss(omit) it.** Moreover, if it already happened, even full rebuilding of the project (and work-area) do not correct the fault. In this case only one method helps : full closing of work-area, reopen it, and full re-building. After that – in OCC v.6.041.1 – all works good. 
**For you pleasure I should say, that in MS Visual Studio – all is MUCH MORE BAD, than in OCC !!!**   : ) :)  :) 
Really : now I CAUSED rapidly rewrite complicated device-data re-calculation program – to replace first it version, written 2-3 years ago on MS Visual Studio 2012 (bought) by another programmer. **Because last month this “powerful MS-Visual Studio made” huge data re-calculation program have became fail in full on Windows-10, and even in Windows-7 computers, probably after next Windows update !!!** As result - we have THE ONLY computer with this important data-treatment (about Earth radiation), the only computer – where were forbidden any update of Windows!!! **Thus we saw all “advantages” of firmware MS Visual Studio!!!** As result – I should fast re-write this program on OCC, at that I have no source code – that programmer with MS Visual Studio leaved us…    
**You can decide yourself - leave this issue opened while, or close it... ?**
**Many thanks for your efforts...**
